src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.html

```diff
@@ -10,8 +10,8 @@ <h1>Student Activity Logs</h1>
           <label for="course-id-dropdown" class="font-weight-bold">Course ID</label>
         </div>
         <div class="col-12">
-          <select id="course-id-dropdown" class="form-control col-8" [(ngModel)]="formModel.courseId">
-            <option *ngFor="let course of courses" (click)="updateSearchButtonDisabled()">
+          <select id="course-id-dropdown" class="form-control col-8" [(ngModel)]="formModel.courseId" (click)="updateSearchButtonDisabled()">
```

Comment:
It still doesn't work for me. On debugging it seems like when the `click` event fires the value of `this.formModel.courseId` is still an empty string, so it fails the if condition check.

Perhaps we can change `click` to `ngModelChange`?

Reply:
Alright! Changed to `ngModelChange`. Also removed the date checks since they are always defaulted to some value on load